plugins/ingest-attachment/src/test/resources/org/elasticsearch/ingest/attachment/test/sample-files/text-in-french.txt

```diff
@@ -0,0 +1 @@
+Allons enfants de la Patrie Le jour de gloire est arrivé. Contre nous de la tyrannie
```

Comment:
hrm, I just copied those over, but it does not seem to be used for testing (gets used for the generic tika testing, that is enough I guess, but was not tested in particlar in a unit test before)